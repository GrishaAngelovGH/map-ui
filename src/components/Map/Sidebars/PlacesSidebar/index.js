import PlacesSidebar from './PlacesSidebar'

export default PlacesSidebar