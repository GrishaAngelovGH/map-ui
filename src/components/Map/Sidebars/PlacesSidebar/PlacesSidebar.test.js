import { render } from '@testing-library/react'

import PlacesSidebar from './PlacesSidebar'

window.matchMedia = window.matchMedia || function () {
  return {
    matches: false,
    addListener: function () { },
    removeListener: function () { }
  }
}

test('should render PlacesSidebar component', () => {
  const view = render(<PlacesSidebar showSidebar />)

  expect(view).toMatchSnapshot()
})