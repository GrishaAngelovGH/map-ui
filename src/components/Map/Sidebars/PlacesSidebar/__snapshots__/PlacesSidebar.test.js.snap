// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PlacesSidebar component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body
    class="modal-open"
    data-rr-ui-modal-open=""
    style="padding-right: 1024px;"
  >
    <div />
    <div
      aria-modal="true"
      class="offcanvas offcanvas-start offcanvas-toggling show"
      role="dialog"
      style="visibility: visible;"
      tabindex="-1"
    >
      <div
        class="offcanvas-header"
      >
        <div
          class="offcanvas-title h5"
        >
          Favorite Places
        </div>
        <button
          aria-label="Close"
          class="btn-close"
          type="button"
        />
      </div>
      <div
        class="offcanvas-body"
      >
        <div
          class="row mb-3 border border-3 rounded shadow p-2 bg-light small"
        >
          <div
            class="col-md-12"
          >
            <div
              class="row"
            >
              <div
                class="col-md-10"
              >
                <p
                  class="fw-bold text-center"
                >
                  Bill Baker's Street Restaurant
                </p>
                <div
                  class="d-flex justify-content-between align-items-center"
                >
                  <p
                    class="p-1 w-50 alert alert-success text-center"
                  >
                    Open
                  </p>
                  <p
                    class="text-secondary"
                  >
                    4.3 
                    <i
                      class="bi bi-star-fill text-warning"
                    />
                     (3037 votes)
                  </p>
                </div>
              </div>
              <div
                class="col-md-2"
              >
                <button
                  class="bi bi-chevron-right btn btn-primary w-100 p-1"
                />
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="col-md-12"
              >
                <p
                  class="m-0 text-secondary d-flex align-items-center"
                >
                  <i
                    class="bi bi-card-text fs-4"
                  />
                  <span
                    class="mx-2"
                  >
                    Contemporary all-day European dining
                  </span>
                </p>
                <p
                  class="m-0 text-secondary d-flex align-items-center"
                >
                  <i
                    class="bi bi-clock fs-4"
                  />
                  <span
                    class="mx-2"
                  >
                    Closes
                     11 PM
                  </span>
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          class="row mb-3 border border-3 rounded shadow p-2 bg-light small"
        >
          <div
            class="col-md-12"
          >
            <div
              class="row"
            >
              <div
                class="col-md-10"
              >
                <p
                  class="fw-bold text-center"
                >
                  Pizza Restaurant Waterloo
                </p>
                <div
                  class="d-flex justify-content-between align-items-center"
                >
                  <p
                    class="p-1 w-50 alert alert-danger text-center"
                  >
                    Closed
                  </p>
                  <p
                    class="text-secondary"
                  >
                    4.3 
                    <i
                      class="bi bi-star-fill text-warning"
                    />
                     (3037 votes)
                  </p>
                </div>
              </div>
              <div
                class="col-md-2"
              >
                <button
                  class="bi bi-chevron-right btn btn-primary w-100 p-1"
                />
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="col-md-12"
              >
                <p
                  class="m-0 text-secondary d-flex align-items-center"
                >
                  <i
                    class="bi bi-card-text fs-4"
                  />
                  <span
                    class="mx-2"
                  >
                    Contemporary all-day European dining
                  </span>
                </p>
                <p
                  class="m-0 text-secondary d-flex align-items-center"
                >
                  <i
                    class="bi bi-clock fs-4"
                  />
                  <span
                    class="mx-2"
                  >
                    Opens
                     11 PM
                  </span>
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          class="row mb-3 border border-3 rounded shadow p-2 bg-light small"
        >
          <div
            class="col-md-12"
          >
            <div
              class="row"
            >
              <div
                class="col-md-10"
              >
                <p
                  class="fw-bold text-center"
                >
                  Leicester Square Restaurant
                </p>
                <div
                  class="d-flex justify-content-between align-items-center"
                >
                  <p
                    class="p-1 w-50 alert alert-success text-center"
                  >
                    Open
                  </p>
                  <p
                    class="text-secondary"
                  >
                    4.3 
                    <i
                      class="bi bi-star-fill text-warning"
                    />
                     (3037 votes)
                  </p>
                </div>
              </div>
              <div
                class="col-md-2"
              >
                <button
                  class="bi bi-chevron-right btn btn-primary w-100 p-1"
                />
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="col-md-12"
              >
                <p
                  class="m-0 text-secondary d-flex align-items-center"
                >
                  <i
                    class="bi bi-card-text fs-4"
                  />
                  <span
                    class="mx-2"
                  >
                    Contemporary all-day European dining
                  </span>
                </p>
                <p
                  class="m-0 text-secondary d-flex align-items-center"
                >
                  <i
                    class="bi bi-clock fs-4"
                  />
                  <span
                    class="mx-2"
                  >
                    Closes
                     11 PM
                  </span>
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          class="row mb-3 border border-3 rounded shadow p-2 bg-light small"
        >
          <div
            class="col-md-12"
          >
            <div
              class="row"
            >
              <div
                class="col-md-10"
              >
                <p
                  class="fw-bold text-center"
                >
                  Mayfair Restaurant
                </p>
                <div
                  class="d-flex justify-content-between align-items-center"
                >
                  <p
                    class="p-1 w-50 alert alert-success text-center"
                  >
                    Open
                  </p>
                  <p
                    class="text-secondary"
                  >
                    4.3 
                    <i
                      class="bi bi-star-fill text-warning"
                    />
                     (3037 votes)
                  </p>
                </div>
              </div>
              <div
                class="col-md-2"
              >
                <button
                  class="bi bi-chevron-right btn btn-primary w-100 p-1"
                />
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="col-md-12"
              >
                <p
                  class="m-0 text-secondary d-flex align-items-center"
                >
                  <i
                    class="bi bi-card-text fs-4"
                  />
                  <span
                    class="mx-2"
                  >
                    Contemporary all-day European dining
                  </span>
                </p>
                <p
                  class="m-0 text-secondary d-flex align-items-center"
                >
                  <i
                    class="bi bi-clock fs-4"
                  />
                  <span
                    class="mx-2"
                  >
                    Closes
                     11 PM
                  </span>
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          class="row mb-3 border border-3 rounded shadow p-2 bg-light small"
        >
          <div
            class="col-md-12"
          >
            <div
              class="row"
            >
              <div
                class="col-md-10"
              >
                <p
                  class="fw-bold text-center"
                >
                  Chelsea Restaurant
                </p>
                <div
                  class="d-flex justify-content-between align-items-center"
                >
                  <p
                    class="p-1 w-50 alert alert-success text-center"
                  >
                    Open
                  </p>
                  <p
                    class="text-secondary"
                  >
                    4.3 
                    <i
                      class="bi bi-star-fill text-warning"
                    />
                     (3037 votes)
                  </p>
                </div>
              </div>
              <div
                class="col-md-2"
              >
                <button
                  class="bi bi-chevron-right btn btn-primary w-100 p-1"
                />
              </div>
            </div>
            <div
              class="row"
            >
              <div
                class="col-md-12"
              >
                <p
                  class="m-0 text-secondary d-flex align-items-center"
                >
                  <i
                    class="bi bi-card-text fs-4"
                  />
                  <span
                    class="mx-2"
                  >
                    Contemporary all-day European dining
                  </span>
                </p>
                <p
                  class="m-0 text-secondary d-flex align-items-center"
                >
                  <i
                    class="bi bi-clock fs-4"
                  />
                  <span
                    class="mx-2"
                  >
                    Closes
                     11 PM
                  </span>
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div />,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
